nixos/tests/nixos-rebuild-target-host.nix

```diff
@@ -0,0 +1,96 @@
+import ./make-test-python.nix ({ pkgs, ... }: let
+  inherit (import ./ssh-keys.nix pkgs) snakeOilPrivateKey snakeOilPublicKey;
+
+  deployerAddress = "192.168.0.1";
+  targetAddress = "192.168.0.2";
```

Comment:
You could get this from the `nodes` module argument instead.
It's available at the root of the test, as well as in the individual configs, and as an argument to `testScript`.
Not sure if `make-test-python.nix` exposes module functionality though. Maybe switch from `handleTest` to `runTest` if you want to keep using a near-top-level `let`?

Reply:
Done :+1: